packages/runtime/src/types.ts

```diff
@@ -85,6 +85,7 @@ export interface DeployedAssetInfo {
   creator: AccountAddress
   txId: string
   confirmedRound: number
+  deleted: boolean
```

Comment:
Can we make this flag optional? An app/asset can only be deleted "once". So deleted flag will only be true when asset/app is deleted from network, and afterwards we don't save any info in checkpoints (as after deletion only optOut is allowed, which is not cp related). 
Not sure about having `deleted:false` in artifacts each time after deploying or updating ASA/SSC.
What do you think?

Reply:
i think we can keep deleted boolean in checkpoints, we have boolean `true` / `false` user can easily know if it's deleted or not, 
and it also keeps checkpoints consistent, if user wants to use them somewhere else, all attributes will be there.